crates/codspeed/src/shared.rs

```diff
@@ -0,0 +1,22 @@
+// !!!!!!!!!!!!!!!!!!!!!!!!
+// !! DO NOT TOUCH BELOW !!
+// !!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
We can remove this decoration, a simple `WARNING: Has to be in sync with runner` will be enough